packages/clerk-js/src/ui/components/SignIn/SignInStart.tsx

```diff
@@ -235,16 +275,39 @@ export function _SignInStart(): JSX.Element {
 }
 
 const InstantPasswordRow = ({ field }: { field?: FormControlState<'password'> }) => {
+  const [autofilled, setAutofilled] = useState(false);
+  const ref = useRef<HTMLInputElement>(null);
+
+  // show password if it's autofilled by the browser
+  React.useLayoutEffect(() => {
+    const intervalId = setInterval(() => {
```

Comment:
💡  Instead of a poller, how about using a mutation observer and possibly avoid the getComputedStyle invocation?

Reply:
trying this out

Reply:
It doesn't seem to fire in that case.